exampleSite/content/home/teaching.md

```diff
@@ -10,7 +10,6 @@ subtitle = ""
 
 # Order that this section will appear in.
 weight = 50
-
```

Comment:
Please revert this commit to be consistent with the blank line before the `+++` in the other `home/` files. Generally, I think it can help make the `+++` separator between TOML and Markdown more obvious to see.